modules/flowable5-engine/src/main/java/org/activiti/engine/impl/bpmn/behavior/MailActivityBehavior.java

```diff
@@ -99,6 +103,24 @@ public void execute(DelegateExecution execution) {
             setCharset(email, charSetStr);
             attach(email, files, dataSources);
 
+            if(LOGGER.isDebugEnabled()) {
```

Comment:
Suggesting a nitpicking formatting change to match the rest of the code base: there should be a space in `if(`; it occurs in each of the files.  Thanks.